config/conventionalcommits-release-rules.js

```diff
@@ -1,6 +1,26 @@
 module.exports = [
+  {
+    type: 'build',
+    release: 'patch',
+  },
+  {
+    type: 'chore',
+    release: 'patch',
+  },
+  {
+    type: 'ci',
+    release: 'patch',
+  },
```

Comment:
I'm pretty concerned about these creating lots of unnecessary churn in our dependencies, especially when we merge dependabot PRs or upgrading our CI pipeline. If something is a chore, affects the build, or ci only, then it shouldn't trigger a release. I think the appropriate change here is to use the `fix` prefix when upgrading a dependency in order to fix functionality rather than changing the definition of `chore`.

Reply:
I agree for `ci` and `refactor`, but I have doubts about `chore`: we mainly use it for bumping module versions, which often produces functional changes and requires a release. Is the solution to use `feat` or `fix` for those bumps, leaving `chore` only for pure "housekeeping" bumps, e.g. those proposed by dependabot?

Reply:
IMO that would be the correct solution here. Bumping module versions that change functionality or fix a bug should be a `fix` or `feat`.

Reply:
OK. This makes sense to me. That said, I'd love to think about ways we could make this limitation more discoverable. As it is it's a footgun that's bit at least three different engineers on the team and I'm sure future new hires will make the same mistake.

Reply:
Yep! I think it'd be good to include a link to this https://www.conventionalcommits.org/en/v1.0.0/#specification in our onboarding documentation, or perhaps in the PR guide (or both).

Reply:
OK, I increased its visibility here: https://www.notion.so/mixmax/Chapter-1-Day-To-Day-Development-073a103967db42eab58710630b4601ba

Reply:
I don't think I have access to that link 😜

Reply:
whoops 🙃